src/App.tsx

```diff
@@ -1,7 +1,21 @@
-import './App.css';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import Main from './pages/Main/Main';
+import Smartphones from './pages/Smartphones/Smartphones';
+import Laptops from './pages/Laptops/Laptops';
+import Viewed from './pages/Viewed/Viewed';
+import './styles/globals.css';
 
 function App() {
-  return <></>;
+  return (
+    <BrowserRouter>
```

Comment:
lets use enum here. 
It's should be separete folder enums, is src folder.
There should be one file enum.ts which will reexport other enums from the folder.
Enum structure: 
`const AppRoute = {
ROOT: '/',
SMARTPHONES: '/smartphones',
...other routes
} as const
` 
Also in this file we should add routes type: 
`
type AppRoute =   (typeof AppRoute)[keyof typeof AppRoute];
`
then export enum with type: 
`
export {AppRoute}
`

Reply:
Modified it

Reply:
It's should be a separete file, but it's OK for now